website/src/components/animation-loop-runner.jsx

```diff
@@ -135,7 +135,7 @@ export default class AnimationLoopRunner extends Component {
     const {pageContext, panel = true, stats, sourceLink} = this.props;
     const name = pageContext && pageContext.exampleConfig && pageContext.exampleConfig.title;
 
-    const notSupported = !this.animationLoop?.isSupported?.();
+    const notSupported = this.animationLoop.isSupported && !this.animationLoop.isSupported();
```

Comment:
Hmm this syntax doesn't work? Wonder if we have more problems then...

Reply:
It is another logic. If `isSupported` doesn't exist then `notSupported` should return false.